helm/reana/README.md

```diff
@@ -31,6 +31,7 @@ This Helm automatically prefixes all names using the release name to avoid colli
 | `components.reana_server.uwsgi.log_5xx`                  | Log only error HTTP requests with status code 5xx. To make this configuration effective `components.reana_server.uwsgi.log_all` must be false. | true |
 | `components.reana_ui.announcement`                       | Announcement message displayed in site top banner                                    | None                                            |
 | `components.reana_ui.enabled`                            | Instantiate the [REANA-UI](https://github.com/reanahub/reana-ui)                     | true                                            |
+| `components.reana_ui.file_preview_size_limit`            | Maximum file size (in bytes) that can be previewed in the web interface              | 5242880                                         |
```

Comment:
```suggestion
| `components.reana_ui.file_preview_size_limit`            | Maximum file size (in bytes) that can be previewed in the web interface. Default value: 5242880 = 5 MiB              | 5242880                                         |
```
What do you think? To give a friendly explaination of the number used as default value, as is done for `kubernetes_jobs_timeout_limit` and `kubernetes_jobs_max_user_timeout_limit`